### PR TITLE
Refactor FB to avoid Passing Post Data

### PR DESCRIPTION
**Pull request by bordoni**

### CODE REVIEW

File: src/Tribe/Aggregator/Errors.php [hunk 1]
```diff
@@ -37,6 +37,7 @@ private function __construct() {
 
 		tribe_register_error( 'core:aggregator:invalid-create-record-type', __( 'An invalid import type was used when trying to create this import record.', 'the-events-calendar' ) );
 		tribe_register_error( 'core:aggregator:invalid-csv-file', __( 'You must provide a valid CSV file to perform a CSV import.', 'the-events-calendar' ) );
+		tribe_register_error( 'core:aggregator:invalid-facebook-token', __( 'We received an Invalid Facebook Token from the Servive.', 'the-events-calendar' ) );
```
Comment (borkweb) [inline src/Tribe/Aggregator/Errors.php]: I think these errors are alphabetized. Can you move this one to the correct location?

Comment (bordoni) [inline src/Tribe/Aggregator/Errors.php]: 😞  I can haha

**Comment by bordoni** [comment]
Fixed

**Comment by borkweb** [comment]
:+1:

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [New.php:125](https://github.com/moderntribe/the-events-calendar/blob/cfa23ffb83c0ee5780e4f48b9bb325330b43cb3d/src/Tribe/Aggregator/Tabs/New.php#L125)

> via jenkins/codesniffer-phpcs

**Pull request merged**